libs/common/src/lib/topbar/topbar.component.ts

```diff
@@ -39,4 +40,8 @@ export class TopbarComponent {
       this.router.navigate([url]);
     });
   }
+
+  bookDemo() {
+    window.open('https://calendly.com/nuclia/30min?back=1&month=2023-10', 'blank', 'noreferrer');
```

Comment:
does that mean it works only for October 2023?

Reply:
no, I tested the link: when passing dates in the past it automatically put the current month, so we don't need to take care of it. But if you think it's safer, I can add a system to update the link according to the current date.